.github/workflows/grpc.yml

```diff
@@ -0,0 +1,45 @@
+name: grpc-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  grpc-testsuite-1_37_1:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: grpc
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      #- name: Install dependencies
+      #  run: pip install --upgrade pip setuptools wheel
+      - uses: actions/checkout@v2
+        with:
+          path: ddtrace
+      #- name: Set env
+      # run: echo ::set-env name=RELEASE_VERSION::${GITHUB_REF#refs/*/v}
+      - uses: actions/checkout@v2
+        with:
+          repository: grpc/grpc
+          ref: master
+          path: grpc
+         # submodules: true
+      #install grpc
+      - name: Install grpc
+        run: pip install grpcio
+      #install grpc tools
+      - name: Install grpc tools
+        run: python -m pip install grpcio-tools
+      - name: Inject ddtrace
+        run: pip install ../ddtrace
+      - name: Install Pytest
+        run: pip install pytest
+      - name: Run Http2 test suite 
+        run: ddtrace-run pytest test/http2_test --continue-on-collection-errors -v
```

Comment:
you might not be running the correct tests here.

I don't see anything in `test/` that is testing the Python library.

they might actually be in here? https://github.com/grpc/grpc/tree/master/src/python/grpcio_tests

Reply:
Ahhh thank you. I was using these directories:
https://github.com/grpc/grpc/tree/master/test/http2_test
https://github.com/grpc/grpc/tree/master/test/distrib/python